vllm_omni/worker/gpu_generation_model_runner.py

```diff
@@ -210,6 +223,8 @@ def _run_generation_model(
             sampling_metadata=self.input_batch.sampling_metadata,
             logits_index=logits_indices,
             sampler=self.sampler,
+            num_reqs=self.input_batch.num_reqs,  # Pass batch size to model
+            tokens_per_req=tokens_per_req,  # Pass per-request token counts
         )
```

Comment:
could we handle the batching logics here? i.e., follows the process of handling multimodal outputs in gpu_ar_model_runner.